src/org/zaproxy/zap/extension/browserView/ZapAddOn.xml

```diff
@@ -1,11 +1,11 @@
 <zapaddon>
 	<name>Browser View</name>
-	<version>4</version>
+	<version>5</version>
 	<status>alpha</status>
 	<description>Adds an option to render HTML responses like a browser</description>
 	<author>ZAP Dev Team</author>
 	<url></url>
-	<changes>Updated for ZAP 2.4</changes>
+	<changes>BrowserPanels size adjusts to parent container.</changes>	
```

Comment:
Regarding the change, might worth rewording it a bit (e.g. `Allow to properly scroll the rendered page`?), this is for the user which most likely doesn't know what the `BrowserPanel` actually is.